app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -78,6 +80,12 @@ class PoliciesFragmentPresenter @Inject constructor(
       supportsLinks = true,
       supportsConceptCards = false
     )
+
+    val websitePattern = Pattern.compile(
+      "(http|ftp|https)://([\\w_-]+(?:(?:\\." +
+        "[\\w_-]+)+))([\\w.,@?^=%&:/~+#-]*[\\w@?^=%&/~+#-])"
+    )
```

Comment:
As indicated in my link in the comment below, we currently linkify using Linkify.WEB_URLS. Why isn't that pattern working in our case?